asab/api/log.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import datetime
+from asab.log import LOG_NOTICE
+
+
+class APIHandler(logging.Handler):
+
+	def __init__(self, level=logging.NOTSET, storage=None):
+		super().__init__(level=level)
+
+		self.level = level
+		self.buffer = storage
+
+	def emit(self, record):
+		if record.name == 'asab.metrics.service' and record.levelno == LOG_NOTICE:
```

Comment:
I would like to see metrics on the webapi :-)

Reply:
Please elaborate :P

Reply:
Remove this `if` ...